lib/src/main/java/com/basistheory/android/service/CardBrandEnricher.kt

```diff
@@ -0,0 +1,215 @@
+package com.basistheory.android.service
+
+class CardBrandEnricher {
+
+    object CardMasks {
+        const val MASK_4_8_12GAPS_19LENGTH = "#### #### #### #######"
+        const val MASK_4_8_12GAPS_16LENGTH = "#### #### #### ####"
+    }
+
+    class CardDetails(
+        var brand: String = "",
+        var identifierRanges: List<Pair<String, String?>>,
+        var validLengths: IntArray,
+        var cvcMask: String = "",
+        var cardMask: String = ""
+    )
+
+    class CardResult(
+        var cardDetails: CardDetails?,
+        var complete: Boolean
+    )
+
+    private val cardBrands = listOf(
+        CardDetails(
+            "visa",
```

Comment:
I would suggest that these brands should be defined in an enum, but that would blow up the events implementation. Can we at least extract them as constants somewhere public so that consumers can see them listed and know what values to expect?